### PR TITLE
ZCS1866 - updating time zone strings (20171222)

### PR DESCRIPTION
**Pull request by mihaipopahtp**

### CODE REVIEW

File: messages/TzMsg_ja.properties [hunk 1]
```diff
@@ -15,136 +15,137 @@
...
-Australia/Darwin = GMT +09:30 \u30c0\u30fc\u30a6\u30a3\u30f3
-Australia/Hobart = GMT +10:00 \u30db\u30d0\u30fc\u30c8
-Australia/Perth = GMT +08:00 \u30d1\u30fc\u30b9
-Australia/Sydney = GMT +10:00 \u30ad\u30e3\u30f3\u30d9\u30e9\u3001\u30e1\u30eb\u30dc\u30eb\u30f3\u3001\u30b7\u30c9\u30cb\u30fc
-America/Sao_Paulo = GMT -03:00 \u30d6\u30e9\u30b8\u30ea\u30a2
-America/Halifax = GMT -04:00 \u5927\u897f\u6d0b\u6a19\u6e96\u6642\uff08\u30ab\u30ca\u30c0\uff09
-America/St_Johns = GMT -03:30 \u30cb\u30e5\u30fc\u30d5\u30a1\u30f3\u30c9\u30e9\u30f3\u30c9
-America/Regina = GMT -06:00 \u30b5\u30b9\u30ab\u30c1\u30e5\u30ef\u30f3
-Etc/GMT+12 = GMT -12:00 \u65e5\u4ed8\u5909\u66f4\u7dda
+Africa/Algiers = GMT+01:00\u897f\u4e2d\u592e\u30a2\u30d5\u30ea\u30ab
```
Comment (silentsakky) [inline messages/TzMsg_ja.properties:18]: is the spaces around offset removed by mistake?
GMT+01:00 should be GMT +01:00

Comment (mihaipopahtp) [reply messages/TzMsg_ja.properties:18]: no, the space is left by mistake in America/Cancun
There should be no space. Correcting this now

**Review by silentsakky — APPROVED** [review]

**Review by fatkudu — APPROVED** [review]

**Comment by fatkudu** [comment]
Looks like there are some merge conflicts now.  Was that last commit (merge branch...) intentional?

**Comment by mihaipopahtp** [comment]
I saw the conflicts before committing.  
The last commit is intentional and ads some leading zeros I initially missed for Chinese and 2 more other languages + one small fix for Arabic I saw while testing. 
I tried to squash all commits into a single one but failed to do so.

I will try and see if I can cancel all commits and do a fresh one with the fully updated files shortly

**Comment by mihaipopahtp** [comment]
ok, I can't figure out how to squash these commits, sorry.
The last one was intentional.

**Comment by fatkudu** [comment]
This is the current state of some commits in this repo:

        N *    2eca3582 [Mihai Claudiu Popa 2018-01-30] (origin/bugfix/ZCS-1866, bugfix/ZCS-1866) Merge branch 'bugfix/ZCS-1866' of https://github.com/Zimbra/zm-timezones into bugfix/ZCS-1866
          |\  
        N | *  654daa42 [Mihai Claudiu Popa 2018-01-30] - fixing leading zero inconsistencies 20180129
        Y * |  b8d47e0c [Mihai Claudiu Popa 2018-01-30] - fixing leading zero inconsistencies 20180129
          |/  
        Y *  706c21a4 [mihaipopahtp 2018-01-29] Update TzMsg_ar.properties
        Y *  7c943edd [Mihai Claudiu Popa 2017-12-22] - updating time zone strings (20171222)
        N | *    f0fa1f66 [Greg Solovyev 2018-01-25] (tag: 8.8.6, origin/master) Merge branch 'release/8.8.6'
          | |\  
          | |/  
          |/|   
        N | *    303ef865 [Greg Solovyev 2017-11-28] (tag: 8.8.5) Merge branch 'release/8.8.5'
          | |\  
        N | | | *  5951a93f [Saket Patel 2017-09-28] (origin/bugfix/ZCS-1417) ZCS-1417 - adding Vietnamese translation - adding ZCS-2592 updates to smart quotes
          | |_|/  
          |/| |   
        N | | | *  7de7002b [Rupali Desai 2017-11-29] (origin/bugfix/ZCS-3524) ZCS-3524 Fixing Timezone string for Astrakhan
          | |_|/  
          |/| |   
          | | | *  351f5e74 [Mihai Claudiu Popa 2017-09-06] (HEAD -> develop, origin/develop, origin/bugfix/ZCS-2592, origin/HEAD) ZCS-2592
          | |_|/  
          |/| |   
          * | |  9baebe33 [Shriram V 2017-11-08] (origin/release/8.8.6) Updated to create the package index

I've marked the commits I think that are needed since develop with a Y and those which are not relevant with an N

I think the root of your problems is that it appears that you started this branch on the
`release/8.8.6` branch instead of the `develop` branch, so the work in your commit `351f5e74` is not included.

I recommend you do something like the following:

        git fetch --all
        git checkout develop

If this `git pull` doesn't make your develop branch the same as `origin/develop`,
you need to force it to be the same as that.  If you need help with that, give me a shout.

        git pull

Create a temporary work branch based on develop

        git flow bugfix start ZCS-1866-2

Start applying the desired commits from your old branch to the new branch:

        git cherry-pick 7c943edd

This will report conflicts.  Resolve those conflicts by editting the files mentioned as being `Unmerged paths:`,
finding the conflict markers, making the necessary changes then doing `git add` for the resolved file.

e.g. the following in `messages/TzMsg_es.properties` has these conflict markers:

        <<<<<<< HEAD
        Pacific/Tongatapu = GMT +13:00 Nuku\u2019alofa
        =======
        Pacific/Tongatapu = GMT +13:00 Nukualofa
        >>>>>>> 7c943edd... - updating time zone strings (20171222)

When everything is resolved, do:

        git cherry-pick --continue

Repeat the process with:

        git cherry-pick 706c21a4
        git cherry-pick 7c943edd
        git cherry-pick b8d47e0c

At this point you should have everything looking good.  Once you are happy with that then do the following to
make the bugfix/ZCS-1866 the same as this branch

        git branch -D bugfix/ZCS-1866  # delete your previous fix branch if it exists
        git checkout bugfix/ZCS-1866   # make a new branch point at the same place as your temporary branch

Push this new branch to origin - this should cause this pull request to be refreshed

        git push -f

        git branch -D bugfix/ZCS-1866-2  # delete the temporary branch

**Comment by mihaipopahtp** [comment]
@fatkudu 
Can you please let me know if these look ok now?
I pulled the current files and compared them against what I have as a final version and they are the same.

**Comment by fatkudu** [comment]
@mihaipopahtp I can't see that anything has changed. and github is still saying that "This branch has conflicts that must be resolved".  Perhaps it would be better for me to try and do this and report to you on anything that I can't immediately resolve?

**Comment by mihaipopahtp** [comment]
@fatkudu 
I think I got it this time. 
I squashed all commits into a single one with all fixes.
Please let me know if you see anything out of place.

**Review by fatkudu — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]
LGTM

**Pull request merged**